imports/ui/components/password/client/password.tests.js

```diff
@@ -12,10 +12,17 @@ import {
 } from 'meteor/accounts-base';
 
 import {
+    LoadController
+} from '../../../modules/load/load';
+
+import {
     sinon
 } from 'meteor/practicalmeteor:sinon';
 
 describe('Password', function() {
+    if (!process.env.TESTING)
+        process.env.TESTING = 1;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '{' and instead saw 'process'.](https://www.codacy.com/app/ddspog/MAM-Seed/file/2739089202/issues/source?bid=3340041&fileBranchId=3353891#l24)